libtelemetry/src/test/java/com/mapbox/android/telemetry/MapboxTelemetryTest.java

```diff
@@ -473,6 +473,38 @@ public void checksSessionIdRotationIntervalNotUpdated() throws Exception {
     assertFalse(notUpdatedSessionInterval);
   }
 
+  @Test
+  public void checksAppUserTurnstileNotQueued() throws Exception {
+    Context mockedContext = obtainNetworkConnectedMockedContext();
```

Comment:
is there a way to avoid mocking external classes (android framework and okhttp) in our tests?